pkgs/top-level/ocaml-packages.nix

```diff
@@ -715,6 +715,8 @@ let
 
     process = callPackage ../development/ocaml-modules/process { };
 
+    ocaml-protoc = callPackage ../development/ocaml-modules/ocaml-protoc { };
```

Comment:
Please keep this list sorted.

Reply:
oh, I've forgotten to update after changing the `protoc`->`ocaml-protoc`